dist/BaseController.d.ts

```diff
@@ -0,0 +1,109 @@
+/**
```

Comment:
All `dist/` files are temporary so that I can test https://github.com/MetaMask/metamask-extension/pull/13865.